src/bindings/python/src/openvino/runtime/opset_utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.runtime import Node
```

Comment:
```suggestion
```
hm, I don't think that this import needed here -- it'll allow users to import Node `from openvino.runtime.opset_utils import Node` that isn't right

Reply:
It's not right and it was unused import, but it was here in `openvino/runtime/opset_utils.py`.
I don't know if somebody uses it incorrectly, but it doesn't hurt to leave it here during the deprecation period
I removed this import in the new location of opset utils